react-native-gutenberg-bridge/RNReactNativeGutenbergBridge.podspec

```diff
@@ -1,18 +1,18 @@
 
 Pod::Spec.new do |s|
   s.name         = "RNReactNativeGutenbergBridge"
```

Comment:
Sorry, I'm picky with names 😁, but why not just `Gutenberg`?

Reply:
This is just the `Bridge` module, not the "whole RN app".
Probably we want to name `Gutenberg` to the whole Gutenberg module (RN App) 😄 

We will see it more clearly when we work on the integration with the WPiOS app in detail.

Reply:
Not sure if relevant but, I'd avoid using the "Gutenberg" generic term just yet. We need to be able to say "bridge" and understand which component we're talking about and ideally how to search for it. Just my 2c.

Reply:
Got it, I think I need to catch up a bit on how the details of integrating work 😁

In that case, `GutenbergBridge` seems reasonable. `RN` and `ReactNative` is saying the same thing twice, and IMO both are redundant since we only have a “bridge” because Gutenberg is RN. This isn’t super important, but the odd naming caught my attention